src/main/kotlin/uk/gov/justice/digital/hmpps/prisonertonomisupdate/activities/ActivitiesService.kt

```diff
@@ -206,6 +204,7 @@ class ActivitiesService(
         incentiveLevel = p.incentiveNomisCode,
         payBand = p.prisonPayBand.nomisPayBand.toString(),
         rate = BigDecimal(p.rate!!).movePointLeft(2),
+        startDate = p.startDate,
```

Comment:
This is the only interesting change - pass the pay rate start date to the NOMIS API. Everything else is just catching up with changes to the API model.